test/dlc_tests/conftest.py

```diff
@@ -169,6 +169,11 @@ def delete_s3_artifact_copy():
     conn.run(f"aws s3 cp --recursive {test_utils.TEST_TRANSFER_S3_BUCKET}/{artifact_folder} $HOME/container_tests")
     conn.run(f"mkdir -p $HOME/container_tests/logs && chmod -R +x $HOME/container_tests/*")
 
+    # Log into ECR if we are in canary context
+    if test_utils.is_canary_context():
+        conn.run(f"$(aws ecr get-login --no-include-email --region "
```

Comment:
Should this use `get-login-password` as in https://docs.aws.amazon.com/deep-learning-containers/latest/devguide/deep-learning-containers-ec2-setup.html ?

Reply:
I'm unable to use this command manually on an EC2 instance -- we might consider updating this documentation

```
$ aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 763104351884.dkr.ecr.us-east-1.amazonaws.com
usage: aws [options] <command> <subcommand> [<subcommand> ...] [parameters]
To see help text, you can run:

  aws help
  aws <command> help
  aws <command> <subcommand> help
aws: error: argument operation: Invalid choice, valid choices are:

batch-check-layer-availability           | batch-delete-image
batch-get-image                          | complete-layer-upload
create-repository                        | delete-lifecycle-policy
delete-repository                        | delete-repository-policy
describe-images                          | describe-repositories
get-authorization-token                  | get-download-url-for-layer
get-lifecycle-policy                     | get-lifecycle-policy-preview
get-repository-policy                    | initiate-layer-upload
list-images                              | put-image
put-lifecycle-policy                     | set-repository-policy
start-lifecycle-policy-preview           | upload-layer-part
get-login                                | help
Error: Cannot perform an interactive login from a non TTY device
```

Reply:
You will need the latest version of awscli https://docs.aws.amazon.com/cli/latest/reference/ecr/get-login-password.html

Reply:
Updated AMI to support this command, good catch and thanks for the suggestion!